build.gradle.kts

```diff
@@ -181,7 +181,8 @@ subprojects {
     }
 
     dependencies {
-        implementation(kotlin("stdlib", kotlinVersion))
+        implementation(kotlin("stdlib-jdk8", kotlinVersion))
```

Comment:
IMHO client libs should depend on `stdlib` only whereas all other ones should be pulling in `jdk8` flavor. Doing this change in the root file will apply it on ALL projects.